readme.txt

```diff
@@ -251,6 +251,7 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 * Fix - Fixes the missing notification on the email and removes the notification when there's none [99979]
 * Fix - Prevent overwriting the start date of a ticket if it was already set [99601]
 * Fix - The redirection to the correct post type URL when site has a plain permalink structure on the buy ticket form [96640]
+* Fix - Make sure the dates to compare if a ticket is in range uses the event time zone as the reference date [99721]
```

Comment:
We could credit evolutionstartup for flagging the situation here (but don't let that hold up merging this work)